prow/plugins/label/label.go

```diff
@@ -71,6 +85,40 @@ type githubClient interface {
 	BotName() string
 }
 
+type AliasInterface interface {
+	hasStatusPrivilege(login string) bool
+}
+
+type Alias struct{}
+
+func (Alias) hasStatusPrivilege(login string) bool {
+	resp, err := http.Get(OWNERS_URL)
```

Comment:
This is ok for now, but we should open an issue for this file to be prefetched and read from disk, instead of having to fetch over HTTP every time the webhook is invoked.

Reply:
Yes, we have a git client in the PluginClient that can just do a Clone(kubernetes/kubernetes) to get this information.